src/__swaps__/screens/Swap/providers/swap-provider.tsx

```diff
@@ -264,24 +264,20 @@ export const SwapProvider = ({ children }: SwapProviderProps) => {
         }
       }
 
-      queryClient.invalidateQueries([
-        // old user assets invalidation (will cause a re-fetch)
-        {
-          queryKey: userAssetsQueryKey({
-            address: parameters.quote.from,
-            currency: nativeCurrency,
-            connectedToHardhat,
-          }),
-        },
-        // new swaps user assets invalidations
-        {
-          queryKey: swapsUserAssetsQueryKey({
-            address: parameters.quote.from as Address,
-            currency: nativeCurrency,
-            testnetMode: !!connectedToHardhat,
-          }),
-        },
-      ]);
+      queryClient.invalidateQueries(
```

Comment:
separated all of the instances of invalidateQueries out because Arrays weren't working for whatever reason.